### PR TITLE
Docker Improvements

### PR DESCRIPTION
**Pull request by wrathagom**
**Proposed changes**:
- Removed java install from all Docker images
- Changed Dockerfile_spacy_duckling to use HTTP version
- Created Tensorflow Dockerfile
- Create sample config for spacy with duckling, with default URL set for compose below
- Created docker-compose file to quickly bring up rasa and duckling side by side.
- moved docker-cloud to root, this is necessary for the DOcker cloud button to work

@milutz can I get a 3rd party review from you :)

**Status (please check what you already did)**:
- [X] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by wrathagom** [comment]
@tmbo do you want me to put back in `ner_duckling` so the tests pass. Or remove the duckling specific tests? Or does Jenkins provide some way to run a docker image and connect to it to test that way?

File: docker/Dockerfile_spacy_sklearn_duckling [hunk 1]
```diff
@@ -0,0 +1,39 @@
+FROM python:3.6-slim
```
Comment (tmbo) [inline docker/Dockerfile_spacy_sklearn_duckling]: is this one different from the dockerfile for spacy_sklearn or is it just using a different config file?

**Comment by tmbo** [comment]
I guess we are ready to merge here right?

**Comment by tmbo** [comment]
Ah ok, the duckling tests still fail.

**Comment by wrathagom** [comment]
sorry @tmbo I've been too busy to get back to this one. I may split it into two PR one that requires test changes and one that doesn't. I'll try to find some time shortly.

**Pull request closed without merging**